pkg/cmd/inspect.go

```diff
@@ -182,3 +304,30 @@ func printDependencies(format string, dependecies *strset.Set) error {
 	}
 	return nil
 }
+
+func getWorkingDirectory() (string, error) {
+	currentDirectory, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	return currentDirectory, nil
+}
+
+func createCamelCatalog() (*camel.RuntimeCatalog, error) {
+	// Attempt to reuse existing Camel catalog if one is present.
+	catalog, err := camel.MainCatalog()
```

Comment:
`camel.DefaultCatalog()` is better. We're removing main.

Reply:
Replaced.